core/settings.py

```
@@ -0,0 +1,654 @@
+    CACHES = values.DictValue(
+        {
+            "default": {
+                "BACKEND": "django_prometheus.cache.backends.redis.RedisCache",
+                "LOCATION": os.getenv("REDIS_CACHE_TOKEN"),
+                "OPTIONS": {
+                    "CLIENT_CLASS": "django_redis.client.DefaultClient",
+                },
+            },
+            "select2": {
+        "BACKEND": "django_redis.cache.RedisCache",
+        "LOCATION": f"{os.getenv("REDIS_CACHE_TOKEN")}/2",
```

Pylint (3.3.7)

[error] Parsing failed: 'f-string: unmatched '(' (git.core.settings, line 303)'

(E0001)